scheduling-manager/src/main/java/com/amazonaws/blox/scheduling/scheduler/engine/StartTask.java

```diff
@@ -15,29 +15,37 @@
 package com.amazonaws.blox.scheduling.scheduler.engine;
 
 import java.util.concurrent.CompletableFuture;
+import lombok.Builder;
 import lombok.Value;
+import lombok.extern.log4j.Log4j2;
 import software.amazon.awssdk.services.ecs.ECSAsyncClient;
 import software.amazon.awssdk.services.ecs.model.StartTaskRequest;
 import software.amazon.awssdk.services.ecs.model.StartTaskResponse;
 
 @Value
+@Builder
+@Log4j2
 public class StartTask implements SchedulingAction {
   private final String clusterArn;
   private final String containerInstanceArn;
   private final String taskDefinitionArn;
+  private final String group;
 
   @Override
   public CompletableFuture<Boolean> execute(ECSAsyncClient ecs) {
-    // TODO: This will probably require setting the task group to the environment ID too
     CompletableFuture<StartTaskResponse> pendingRequest =
         ecs.startTask(
             StartTaskRequest.builder()
                 .cluster(clusterArn)
                 .containerInstances(containerInstanceArn)
                 .taskDefinition(taskDefinitionArn)
+                .group(group)
+                .startedBy("blox")
```

Comment:
constant

Reply:
It's a single usage of an unimportant string in a temporary class. I'll extract it, but I'm not convinced it's helpful.